UserTools/Factory/Factory.cpp

```diff
@@ -102,13 +102,14 @@ if (tool=="ANNIEEventBuilder") ret=new ANNIEEventBuilder;
 if (tool=="MRDDataDecoder") ret=new MRDDataDecoder;
 if (tool=="PrintADCData") ret=new PrintADCData;
 if (tool=="ClusterFinder") ret=new ClusterFinder;
+if (tool=="PrintRecoEvent") ret=new PrintRecoEvent;
 if (tool=="RunValidation") ret=new RunValidation;
 if (tool=="AmBeRunStatistics") ret=new AmBeRunStatistics;
 if (tool=="SimpleTankEnergyCalibrator") ret=new SimpleTankEnergyCalibrator;
 if (tool=="BeamClusterPlots") ret=new BeamClusterPlots;
 if (tool=="MrdPaddleEfficiencyPreparer") ret=new MrdPaddleEfficiencyPreparer;
 if (tool=="MrdPaddleEfficiencyCalc") ret=new MrdPaddleEfficiencyCalc;
 if (tool=="LoadRawData") ret=new LoadRawData;
-if (tool=="ClusterClassifiers") ret=new ClusterClassifiers;
```

Comment:
again did you mean to delete this tool?

Reply:
No, I didn't mean to delete the ClusterClassifiers tool from the Factory, sorry.